public/app/features/search/page/components/sorting.ts

```diff
@@ -0,0 +1,35 @@
+import { SelectableValue } from '@grafana/data';
+import { config } from '@grafana/runtime';
+
+const sortFields = [
+  { name: 'views_total', display: 'Views total' },
+  { name: 'views_last_30_days', display: 'Views 30 days' },
+  { name: 'errors_total', display: 'Errors total' },
+  { name: 'errors_last_30_days', display: 'Errors 30 days' },
+];
+
+// This should eventually be filled by an API call, but hardcoded is a good start
+export async function getSortOptions(): Promise<SelectableValue[]> {
+  const opts: SelectableValue[] = [
+    { value: 'name_sort', label: 'Alphabetically (A-Z)' },
```

Comment:
consistency: `sortFields` defined on line 4 don't have the `_sort` suffix. Should they?

Reply:
`name_sort` is a special field in the index that is only used only for sorting.  We can't used "name" directly because it is tokenized for better search :) (Ie breaks on spaces etc)

The other numeric fields do not have a `_sort` suffix because we can sort by the numeric value directly.

The real question here is if we should expose the internal field names as the query params.  For simplicity, I think matching names is the easiest

Reply:
Ohhh, alright. I guess it will be better once we have the API call returning all possible sort options + their metadata, right now this combined with `getSortFieldDisplayName` logic was pretty hard to understand :D

Reply:
![2022-05-21_21-44-49 (1)](https://user-images.githubusercontent.com/705951/169679117-d077d610-cdbf-4a3e-aec8-b300c89e80c8.gif)

I kept everything in this one sorting.ts function... for now that seems easier than a new API split between enterprise/oss repos, especially as we figure out the real indexing strategy